sources/srd.py

```diff
@@ -170,1082 +170,11020 @@ def __init__(self):
+                         "the steed takes on a form that you choose: a warhorse, a pony, " +
+                         "a camel, an elk, or a mastiff. (Your GM might allow other " +
+                         "animals to be summoned as steeds.) The steed has the " +
+                         "statistics of the chosen form, though it is a celestial, fey, " +
+                         "or fiend (your choice) instead of its normal type. " +
+                         "Additionally, if your steed has an Intelligence of 5 or less, " +
+                         "its Intelligence becomes 6, and it gains the ability to " +
+                         "understand one language of your choice that you speak. Your " +
+                         "steed serves you as a mount, both in combat and out, and you " +
+                         "have an instinctive bond with it that allows you to fight as a " +
+                         "seamless unit. While mounted on your steed, you can make any " +
+                         "spell you cast that targets only you also target your steed. " +
+                         "When the steed drops to 0 hit points, it disappears, leaving " +
+                         "behind no physical form. You can also dismiss your steed at " +
+                         "any time as an action, causing it to disappear. In either " +
+                         "case, casting this spell again summons the same steed, " +
+                         "restored to its hit point maximum. While your steed is within " +
+                         "1 mile of you, you can communicate with it telepathically. You " +
+                         "can't have more than one steed bonded by this spell at a time. " +
+                         "As an action, you can release the steed from its bond at any " +
+                         "time, causing it to disappear. ",
+                         at_higher_levels="")
+
+
+class FindTraps(spells.Spell):
+    """
+    Find Traps Spell
+    SRD p. 145
+    Generated
+    """
+
+    def __init__(self):
+        super().__init__(name="Find Traps",
+                         spell_lists=[SpellLists.DIVINE, SpellLists.PRIMAL],
+                         level=2,
+                         school=SpellSchools.Divination,
+                         spell_range="120 feet",
+                         verbal_components=True,
+                         somatic_components=True,
+                         material_components_list=None,
+                         duration="Instantaneous",
+                         description="You sense the presence of any trap within range that is within " +
+                         "line of sight. A trap, for the purpose of this spell, includes " +
+                         "anything that would inflict a sudden or unexpected effect you " +
+                         "consider harmful or undesirable, which was specifically " +
+                         "intended as such by its creator. Thus, the spell would sense 
```

Comment:
tables are a bit dicey

Reply:
This is a big one, I fixed the rest locally but the parser will need to work on this